pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -310,10 +319,27 @@ func TestOpenShiftClusterStaticValidateClusterProfile(t *testing.T) {
 			},
 			wantErr: "400: InvalidParameter: properties.clusterProfile.version: The provided version 'invalid' is invalid.",
 		},
+		{
+			name: "leading digit domain invalid",
+			modify: func(oc *OpenShiftCluster) {
+				oc.Properties.ClusterProfile.Domain = "4k7f9clk"
+			},
+			wantErr: "400: InvalidParameter: properties.clusterProfile.domain: The provided domain '4k7f9clk' is invalid.",
```

Comment:
This test is to ensure that the behaviour doesn't change for new clusters: we still do not allow digits in the beginning of the domain name for new clusters.